FlightPlanProject/UI/TabPages.cs

```diff
@@ -0,0 +1,146 @@
+
+
+using KSP.Sim.impl;
+
+using FlightPlan.KTools.UI;
+
+namespace FlightPlan;
+
+public class BasePageContent : PageContent
+{
+    public BasePageContent()
+    {
+        this.main_ui = FlightPlanUI.Instance;
+        this.plugin = FlightPlanPlugin.Instance;
+    }
+    protected FlightPlanUI main_ui;
+    protected FlightPlanPlugin plugin;
+
+
+    protected PatchedConicsOrbit orbit => main_ui.orbit;
+    protected CelestialBodyComponent referenceBody => main_ui.referenceBody;
+
+    public virtual string Name => throw new NotImplementedException();
+
+    public bool isRunning => false;
+
+
+    bool ui_visible;
+    public bool UIVisible { get => ui_visible; set => ui_visible = value; }
+
+    public virtual bool isActive => throw new NotImplementedException();
+
+    public virtual void onGUI()
+    {
+        throw new NotImplementedException();
+    }
+}
+
+public class OwnshipManeuversPage : BasePageContent
+{
+    public override string Name => "Own Orbit";
+
+    public override bool isActive => true;
+
+    public override void onGUI()
+    {
+        FPStyles.DrawSectionHeader("Ownship Maneuvers");
+        BurnTimeOption.Instance.OptionSelectionGUI();
```

Comment:
I moved the selection of time in each tabs. not sure if it's a good idea. you can place it back near to target selection if you prefers